libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -324,6 +324,7 @@ string IRGenerator::generateFunction(FunctionDefinition const& _function)
 	return m_context.functionCollector().createFunction(functionName, [&]() {
 		m_context.resetLocalVariables();
 		Whiskers t(R"(
+			/// @ast-id <astID>
```

Comment:
Created this new annotation - opinions?

Reply:
What will this be used for? The function entry points debug info?

Reply:
Yes, but we could also use this for e.g. tracking from which solidity variable a yul variable originates.